circuits/src/zk_gadgets/comparators.rs

```diff
@@ -1,23 +1,27 @@
 //! Groups gadgets for binary comparison operators
+//!
+//! Some gadgets are implemented for both single-prover and multi-prover
+//! however, some gadgets are only implemented for single-prover circuits.
+//! This is done when the gadget is inefficient and unneeded in an MPC circuit.
+//! Or, for example with `EqZero` gadget, if the gadget would leak privacy.
```

Comment:
re: our discussion

let's move this doc comment to e.g. `circuits/src/zk_gadgets/mod.rs`, and have it call out that the boundary for gadgets defined in this repo vs `mpc-jellyfish` is those that can't be defined generically b/t single and multi-prover and/or those that are high-level, application-specific gadgets for the Renegade protocol.